### ISSUE TITLE
Editorial Meeting agenda 26/05/2017

### ISSUE THREAD

**Issue by acrymble** [programminghistorian/jekyll#450]
Apologies: @ianmilligan1 @amandavisconti @JMParr @mariajoafana 
Present: @acrymble @walshbr @jerielizabeth @vgayolrs @arojascastro @fredgibbs @mdlincoln @wcaleb 

Sorry, forgot to set this ticket up. Suggested topics (please suggest others):

* Thanks to Caleb & Fred
* Ian Sabbatical (till October)
* ~~Should we separate board from editorial team #379 (has been postponed twice)~~
* Twitterbot #402 ; #418 
* Site redesign #448 #447 #446 #445 #444 #443 #429 #428 #427 #426 #406 #405 #404 #341 #321 #303 #130 

That last one is likely to be a fairly big point, so that might be all we can manage at the moment?

**Comment by mdlincoln** [comment]
Agreed that the site redesign conversation will take most of our time.

Am I missing a major email re: #379, or has nothing been written down about this proposed separation of the board? At the risk of delaying this conversation further, I'd feel more equipped to discuss it if there were some articulated positions to work from. That's not a discussion I want to go in to blind.

**Comment by ianmilligan1** [comment]
Again apologies that I'll miss this (I'll be on the road, sigh). I agree with @mdlincoln that we should postpone #379 and have some concrete discussion, probably over e-mail, especially as it involves all of us.

**Comment by walshbr** [comment]
I'll be there!

**Comment by acrymble** [comment]
Ok, will postpone #379. This was originally raised by @fredgibbs. Perhaps he could flush out a vision in the ticket so we can discuss it in future?

**Comment by acrymble** [comment]
Note to decide on PDF/Markdown version of record format #428 before @wcaleb does more work on it. (next call)

**Comment by acrymble** [comment]
Minutes:

@walshbr has been tasked with making the twitter bot live!

As a group we went through the new design at http://fredgibbs.net/jekyll and a number of decisions have been made and new issues created as a result. All are documented through the github issues on this repo.

We discussed the possible need for more active editors, which we will discuss at the next meeting. #460 / #461 

The Spanish team reported good progress on translations.
